### ISSUE TITLE
Add issues_url to agent metadata json?

### ISSUE THREAD

**Issue by optiz0r** [choria-io/go-choria#701]
When building a package module, PDK issues the following warning: 
```
pdk (WARN): This module is missing the following fields in the metadata.json: issues_url. These missing fields may affect the visibility of the module on the Forge.
```
It would be good (although definitely not a high priority) to allow this field in the agent JSON, and copy the value across to the PDK metadata.json.

**Comment by ripienaar** [comment]
yup, this is infurating, they moan about things they say are optional.  At least after a previous ticket they are only warning and not refusing to build
